dali/ft/filecopy.cpp

```diff
@@ -397,10 +397,31 @@ bool FileTransferThread::launchFtSlaveCmd(const SocketEndpoint &ep)
     }
     else
     {
-        setDafsEndpointPort(connectEP);
-        if (connectEP.isNull())
-            return false;
-        socket.setown(connectDafs(connectEP, 5000));
+        if (!isContainerized())
+        {
+            setDafsEndpointPort(connectEP);
```

Comment:
NB: this is reall a cosmetic change, previous code wouldn't actually have set EP in containerized mode, because in practice setDafsEndpointPort only set port if not already set.